goto/the_real_goto.py

```diff
@@ -51,42 +51,42 @@ def run_command(magic, command, args):
         return commands.usage()
 
     if command == 'add':
-        return commands.add(magic, args)
+        return commands.add(magic, command, args)
 
     if command == 'update':
-        return commands.update(magic, args)
+        return commands.update(magic, command, args)
 
     if command == 'rm':
-        return commands.rm(magic, args)
+        return commands.rm(magic, command, args)
 
     if command == 'show':
-        return commands.show(magic, args)
+        return commands.show(magic, command, args)
 
     if command == 'copy':
-        return commands.copy(magic, args)
+        return commands.copy(magic, command, args)
 
     if command == 'list':
-        return commands.list(magic, args)
+        return commands.list(magic, command, args)
 
     if command == 'subl':
-        return commands.subl(magic, args)
+        return commands.subl(magic, command, args)
 
     if command == 'vscode':
-        return commands.vscode(magic, args)
+        return commands.vscode(magic, command, args)
 
     if command in ['intellij', 'idea']:
-        return commands.intellij(magic, args)
+        return commands.intellij(magic, command, args)
 
-    if '-o' in sys.argv or '--open' in sys.argv or 'open' in sys.argv:
-        return commands.open(magic, args)
+    if command in ['-o', '--open', 'open']:
+        return commands.open(magic, command, args)
 
     if command == 'cd':
-        return commands.cd(magic, args)
+        return commands.cd(magic, command, args)
 
     if command in ['mv', 'rename']:
         return commands.rename(magic, command, args)
 
-    return commands.default(magic, command)
+    return commands.default(magic, command, args)
```

Comment:
Ref the comment in `default.py` I believe this should be treated a special case, where the outcome is:

set command to `None` 
and send all sys.argv in the args parameter (except the jfile and scriptname)

Reply:
How about
```py
    args = [command] + args
    return commands.default(magic, args)
```
vs
```py
    command = None
    return commands.default(magic, command, args)
```
I like the first one better

Reply:
First one seems to be cleaner perhaps if adding an explaining comment